core/src/components/radio-group/radio-group.tsx

```diff
@@ -317,11 +318,11 @@ export class RadioGroup implements ComponentInterface {
     }
 
     return (
-      <div class="radio-group-top">
-        <div id={helperTextId} class="helper-text" aria-live="polite">
+      <div class="radio-group-top" part="top">
```

Comment:
```suggestion
      <div class="radio-group-top">
```

Let's not add this part right now since we don't expose this wrapper for checkbox or select.